cypress/support/commands.js

```diff
@@ -52,8 +52,8 @@ Cypress.Commands.add('openItemThroughMenuAccount', (menuAccountLabel) => {
 });
 
 // Custom command to perform login using environment variables
-Cypress.Commands.add('deleteProduct', (productName, path) => {
-  deleteProduct(productName, path);
+Cypress.Commands.add('deleteProduct', (productName, columnName, path) => {
```

Comment:
```suggestion
Cypress.Commands.add('deleteProduct', (path, productName, columnName = 'name') => {
```
Sugestão: como várias chamadas utilizam o "name" como valor, assim deixamos o name como opcional
** caso aceite a sugestão, alterar os métodos chamados, para usar o path como primeiro parametro